samples/Actor/DemoActor/Program.cs

```diff
@@ -5,42 +5,21 @@
 
 namespace DaprDemoActor
 {
-    using Dapr.Actors.AspNetCore;
-    using Microsoft.AspNetCore;
     using Microsoft.AspNetCore.Hosting;
-    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
 
-    /// <summary>
-    /// Class for host.
-    /// </summary>
     public class Program
     {
-        /// <summary>
-        /// Entry point.
-        /// </summary>
-        /// <param name="args">Arguments.</param>
         public static void Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        /// <summary>
-        /// Creates a IWebHostBuilder.
-        /// </summary>
-        /// <param name="args">Arguments.</param>
-        /// <returns>IWebHostBuilder instance.</returns>
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>()
-                .ConfigureServices(services => 
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
```

Comment:
note: our examples were using some outdated code. Projects based on 3.1 or 5.0 will have `IHostBuilder` here not `IWebHostBuilder`.